packages/database-editor/src/components/ImportWizard/pages/CreationPage.css

```diff
@@ -1,3 +1,14 @@
 .table-creation .table-header {
   font-weight: 600;
+  width: 100px;
+}
+
+.table-header.table-name {
+  width: 200px;
+}
+
+.table-creation td {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
 }
```

Comment:
Just as always think about that this styles are global. So even if this class names are from you, maybe you could scope them a bit more. e.g.:
```css
.table-creation 
  .table-header {
    font-weight: 600;
    width: 100px;
    &.table-name {
      width: 200px;
    }
  }
  td {
    text-overflow: ellipsis;
    overflow: hidden;
    white-space: nowrap;
  }
}
```